.eslintrc.json

```diff
@@ -1,8 +1,11 @@
 {
   "root": true,
-  "extends": [
-    "plugin:github/recommended",
-    "plugin:github/browser",
-    "plugin:github/typescript"
-  ]
+  "extends": ["plugin:github/recommended", "plugin:github/browser", "plugin:github/typescript"],
```

Comment:
? does this autoformat differently post-update? I've usually seen these one array value per line like the deleted diff has

Reply:
Probably because Prettier updated in this PR? not sure 🤷